src/os/im/importer.js

```diff
@@ -532,18 +615,42 @@ os.im.Importer.prototype.executeMapping = function(item) {
         goog.log.error(os.im.Importer.LOGGER_,
             'Error applying mapping "' + m.field + ' > ' + m.getLabel() + '"! ', e);
       }
+
+      const toField = m.toField || undefined;
+      if (toField && os.im.Importer.isEllipseMapping(m)) {
+        if (this.mappingFailCount[toField] == undefined) {
+          this.mappingFailCount[toField] = 0;
+        }
+        const value = item.get(toField);
+        if (value == undefined || value == '') {
+          this.mappingFailCount[toField] = this.mappingFailCount[toField] + 1;
+        }
+      }
     }
   }
 };
 
 
+/**
+ * Returns if the mapping is for ellipses
+ * @param {os.im.mapping.IMapping} mapping
+ * @return {boolean}
+ */
+os.im.Importer.isEllipseMapping = function(mapping) {
+  const id = mapping.getId();
+  return id == os.im.mapping.RadiusMapping.ID || id == os.im.mapping.OrientationMapping.ID ||
+    id == os.im.mapping.SemiMajorMapping.ID || id == os.im.mapping.SemiMinorMapping.ID;
+};
+
+
 /**
  * @inheritDoc
  */
 os.im.Importer.prototype.reset = function() {
   this.stop();
   this.parser.cleanup();
   this.data_ = [];
+  this.totalCount = 0;
```

Comment:
does the mappingFailCount need to be cleared?

Reply:
For the reset function, probably not. I was just putting it (almost) everywhere the data was reset to avoid double-counting any features.